alica_engine/src/engine/constraintmodul/ConditionStore.cpp

```diff
@@ -14,16 +14,6 @@
 
 namespace alica
 {
-namespace
-{
-std::ostream& operator<<(std::ostream& out, const VariableGrp& vg)
```

Comment:
This change is most intriguing one. This function is in anonymous namespace but `g++` compiles it and leaves it in object file; and in `alica_engine/src/engine/constraintmodul/ConditionStore.cpp:115` it (somehow) links together. (I tried to reproduce it behaviour with much simple example but linker complains)

Reply:
Them being in the same compilation unit, this seems fine.

Reply:
hm... now I see, very strange. Initially clang complains about unused function first.

Reply:
Some connection to the functions in `Output.h` maybe...

Reply:
Not sure, previous implementation was for 2 concrete types which not matches to this.